landscape/prod/maps/sites.map

```diff
@@ -2005,6 +2006,7 @@ _/univcomp content ;
 _/univcomputer content ;
 _/universe content ;
 _/ur content ;
+_/urbanclimate/wp-assets/  content ;
```

Comment:
Trailing slash is not needed. Not sure if it's detrimental but to maintain consistency should be removed.